collector/hyperv.go

```diff
@@ -92,6 +92,22 @@ type HyperVCollector struct {
 	AdapterFramesDropped  *prometheus.Desc
 	AdapterFramesReceived *prometheus.Desc
 	AdapterFramesSent     *prometheus.Desc
+
+	// Win32_PerfRawData_Counters_HyperVVirtualStorageDevice
+	VMStorageErrorCount            *prometheus.Desc
+	VMStorageQueueLength  	       *prometheus.Desc
+	VMStorageReadBytesPersec       *prometheus.Desc
```

Comment:
Drop the `Persec` at the end of the names here

Reply:
Done!